### PR TITLE
Include some basic tests for Regexp to test POSIX bracket expressions for unicode 8.0 / 9.0

### PR DESCRIPTION
**Pull request by submitnine**
I included some basic tests for the POSIX bracket expressions functionality in the Regexp class. The tests will check (depend on the ruby verion) about compatibility of unicode 9.0.

### CODE REVIEW

File: language/regexp_spec.rb [hunk 1]
```diff
@@ -167,4 +167,31 @@
...
+
+      char_uppercase = "\u{104B0}" # OSAGE CAPITAL LETTER A
+      /[[:upper:]]/.match(char_lowercase).to_s.should == nil
+    end
+
+    it "supports handling unicode 8.0 characters with POSIX bracket expressions" do
+      char_lowercase = "\u{A7B5}" # LATIN SMALL LETTER BETA
+      /[[:lower:]]/.match(char_lowercase).to_s.should == char_lowercase
+
+      char_uppercase = "\u{A7B4}" # LATIN CAPTIAL LETTER BETA
```
Comment (eregon) [inline language/regexp_spec.rb]: Typo: Capital

**Comment by eregon** [comment]
Merged, thank you for your contribution!
